src/pages/Background/index.js

```diff
@@ -1,2 +1,66 @@
+//Even while development.. extension need to be reloaded for changes here to come into effect
+import { closest, distance } from 'fastest-levenshtein';
+
 console.log('This is the background page.');
+
 console.log('Put the background scripts here.');
+
+console.log('new line by hamza');
+
+//fetch('http://localhost:5000').then((res) => res.json()).then((res) => { console.log(JSON.stringify(res)) })
```

Comment:
lots of console log, remove them, not good in production

Reply:
yet to remove all this.